stable/metrics-server/values.yaml

```diff
@@ -36,3 +36,9 @@ tolerations: []
 affinity: {}
 
 replicas: 1
+
+# tlsConfig:
+#   caFile:   /etc/kubernetes/certs/ca.crt
+#   certFile: /etc/kubernetes/certs/kubelet.crt
+#   keyFile:  /etc/kubernetes/certs/kubelet.key
+#   insecureSkipVerify: true
```

Comment:
Make sure you take the comment and config on line 27-28 into consideration too, as this would replace that warning.